ethereum/contracts/Getters.sol

```diff
@@ -53,4 +57,4 @@ contract Getters is State {
     function nextSequence(address emitter) public view returns (uint64) {
         return _state.sequences[emitter];
     }
-}
\ No newline at end of file
+}
```

Comment:
Please configure your editor to insert a new line at the end of files for consistent diffs. e.g. https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
sometimes I wonder why this isn't the default in VS code 🤔

Reply:
Perhaps we should consider committing some vscode settings. Maybe after we do prettier config.

Reply:
I believe the new files are adding that new line (that's the diff you see). Should we remove the new line to make the diffs consistent?

Reply:
Ah, perhaps that’s it (can’t easily see from my phone 😛). But no, having them in there is the way.

Reply:
Yeah we should add the newline at the end otherwise the last line is not a real line! (https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)

Some tools consider files with a missing newline at the end corrupted, though modern tools tend to just ignore it